src/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -21,7 +21,9 @@ def create(data,
            max_iterations=0,
            verbose=True):
     """
-    Create a :class:`OneShotObjectDetector` model.
+    Create a :class:`OneShotObjectDetector` model. Note: The One Shot Object Detector
+    is currently in beta, as we make the toolkit faster and even more performant.
```

Comment:
I don't think you need to say things like "stay tuned etc." This is currently in beta is more than enough.

Reply:
Done.